app/views/topical_events/show.html.erb

```diff
@@ -71,14 +71,14 @@
             border_top: 2,
           } %>
         </div>
-        
+
```

Comment:
minor: do we want to commit these formatting changes?

Reply:
Looks like my text editor automatically removed these unneeded spaces.  It surprises me that we don't lint views 🤔 , as this would've been picked up by a linter before it were previously committed.